### PR TITLE
Implement list command

### LINKED ISSUE #104: As a wedding planner, I want to be able to list all the weddings
so that I can see all the weddings I am planning

### PR DESCRIPTION
**Pull request by siriousguy**
Fixes #104

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListWeddingCommand.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public static final String MESSAGE_EMPTY = " Your address book is empty";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredWeddingList(PREDICATE_SHOW_ALL_WEDDINGS);
+        int numberOfWeddings = model.getWeddingBook().getWeddingList().size();
+        String numberOfEvents = ", there are currently " + numberOfWeddings + " weddings in your address book";
+        if (numberOfWeddings == 0) {
+            return new CommandResult(MESSAGE_EMPTY);
```
Comment (siriousguy) [inline src/main/java/seedu/address/logic/commands/ListWeddingCommand.java:25]: Will keep trying to write a test case for this

File: src/main/java/seedu/address/ui/WeddingListPanel.java
Comment (siriousguy) [inline src/main/java/seedu/address/ui/WeddingListPanel.java:1]: Testcases seem to have a bit of problems

File: src/main/java/seedu/address/logic/commands/ListWeddingCommand.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * Lists all the weddings in the address book to the user.
+ */
+public class ListWeddingCommand extends Command {
+
+    public static final String COMMAND_WORD = "list-wedding";
+
+    public static final String MESSAGE_SUCCESS = "Listed all weddings";
+    public static final String MESSAGE_EMPTY = " Your address book is empty";
```
Comment (zzibo) [inline src/main/java/seedu/address/logic/commands/ListWeddingCommand.java:16]: Looks great but I think this should be wedding book

File: src/main/java/seedu/address/logic/commands/ListWeddingCommand.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredWeddingList(PREDICATE_SHOW_ALL_WEDDINGS);
+        int numberOfWeddings = model.getWeddingBook().getWeddingList().size();
+        String numberOfEvents = ", there are currently " + numberOfWeddings + " weddings in your address book";
```
Comment (zzibo) [inline src/main/java/seedu/address/logic/commands/ListWeddingCommand.java:23]: similarly here! maybe can consider renaming wedding book to something else

File: src/main/java/seedu/address/ui/WeddingCard.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    private Label id;
+    @FXML
+    private Label venue;
+    @FXML
+    private Label dateTime;
+
+    /**
+     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     */
+    public WeddingCard(Wedding wedding, int displayedIndex) {
```
Comment (zzibo) [inline src/main/java/seedu/address/ui/WeddingCard.java:40]: well written UI!

File: src/test/java/seedu/address/ui/WeddingListPanelTest.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import javafx.collections.ObservableList;
+import javafx.scene.Parent;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import seedu.address.model.wedding.Datetime;
+import seedu.address.model.wedding.Venue;
+import seedu.address.model.wedding.Wedding;
+import seedu.address.model.wedding.WeddingName;
+
+public class WeddingListPanelTest {
```
Comment (zzibo) [inline src/test/java/seedu/address/ui/WeddingListPanelTest.java:16]: Great to see test for JavaFX Ui

File: src/test/java/seedu/address/ui/WeddingCardTest.java [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+import seedu.address.model.wedding.Venue;
+import seedu.address.model.wedding.Wedding;
+import seedu.address.model.wedding.WeddingName;
+
+import java.lang.reflect.Field;
+import java.util.concurrent.CountDownLatch;
+
+public class WeddingCardTest {
+
+    private static final Wedding SAMPLE_WEDDING = new Wedding(new WeddingName("Alice & Bob"),
```
Comment (zzibo) [inline src/test/java/seedu/address/ui/WeddingCardTest.java:19]: Maybe can consider abstracting sample weddings into another file

**Review by zzibo — COMMENTED** [review]
LGTM! Great to see well written test especially for the JavaFX UI, however there are a few naming bug to be fixed!

File: src/main/java/seedu/address/logic/commands/ListWeddingCommand.java (hunk 3 above)
Comment (siriousguy) [reply src/main/java/seedu/address/logic/commands/ListWeddingCommand.java]: Thank you! Naming has been updated accordingly

**Comment by ywllowsensor** [comment]
LGTM. Even though the CodeCov does not pass, this PR will be merged as majority of the changes involve GUI updates, which are cumbersome to test.

**Pull request merged**